tests/dummy/app/pods/docs/about/template.md

```diff
@@ -0,0 +1,107 @@
+# What this addon is
+
+Let's start with some glossary.
+
+
```

Comment:
I thinks something like this has to be part of the introduction! When looking at the `Why cucumber` page I wondered how anyone who does not know what Cucumber or BDD is should understand it. I think that would help!

Reply:
Done.